flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/JobAutoScalerImpl.java

```diff
@@ -101,6 +101,11 @@ public boolean scale(FlinkResourceContext<? extends AbstractFlinkResource<?, ?>>
                     metricsCollector.updateMetrics(
                             resource, autoScalerInfo, ctx.getFlinkService(), conf);
 
+            if (collectedMetrics.getMetricHistory().isEmpty()) {
+                autoScalerInfo.replaceInKubernetes(kubernetesClient);
```

Comment:
nit: Does it worth to add a comment here as well, like:
```
// We have metrics yet, but we are not ready for scaling.
```

Do I understand correctly the intent of the code?

Reply:
At this point we have no metrics, so there is nothing to do. What you commented actually happens a bit down in the code and has a similar comment :) 

This piece of code was deleted in a previous commit I just restored it and added a test to make sure we dont accidentally remove/break it.